content/docs/get-started/install/migrating-2.0.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Migrating to Pulumi 2.0
+meta_desc: This page provides instructions for migrating to Pulumi 2.0
+no_on_this_page: true
+---
+
+## Migrating to 2.0
+
+Upgrading to 2.0 is simple. First, you will download the 2.0 CLI, which is required to use the 2.0 SDK. Then, update each of your Pulumi programs to utilize the new SDK. If you're using Javascript or Typescript, you'll also need to ensure you're using invokes asynchronously. We provide detailed instructions on each of these steps below.
+
+## Install Pulumi 2.0 CLI
+
+If you are upgrading to a beta release, obtain the latest binaries from the [versions]({{< relref "versions" >}}) page. Then, follow the [manual installation]({{< relref "./#manual-installation" >}}) steps for your operating system.
+
+## Update CLI Scripts
+
+Previously, the Pulumi CLI would assume `--yes` when used in non-interactive mode (i.e. when `pulumi` had its output piped to another process). You will now need to explicitly pass in `--yes` in those scenarios. This affects:
+
+* `pulumi destroy`
+* `pulumi new`
+* `pulumi refresh`
+* `pulumi up`
+
+## Update Your Pulumi Programs
+
+{{< chooser language "javascript,typescript,python,go,csharp" >}}
+
+{{% choosable language "javascript,typescript" %}}
+
+### Update Dependencies
+
+```bash
+npm install @pulumi/pulumi@^2.0.0
+```
+
+### Remove synchronous invokes
+
+Pulumi 2.0 no longer supports synchronous invokes. As a result, using the return values of any resource `get` operations will need to be made asynchronous. Similarly, the use of `getOutputSync` and `requireOutputSync` are no longer supported and you should move to using the `Output`-based versions of `getOutput` and `requireOutput`.
```

Comment:
```suggestion
Pulumi 2.0 no longer supports synchronous invokes. As a result, the return value of all `getSomething` operations are now `Promise<Something>` values, instead of `Something` values, and should be processed using `.then`, `async/await` or by passing the results to `pulumi.output()`. Similarly, the the `StackReference` operations `getOutputSync` and `requireOutputSync` are no longer supported and you should move to using the `Output`-based versions of `getOutput` and `requireOutput`.
```